docs/sources/alerting/fundamentals/alert-rules/_index.md

```diff
@@ -95,25 +95,27 @@ In this setup, the distributed architecture can provide high-availability and fa
+| Create alert rules<wbr /> that query [data sources supporting Alerting](#supported-data-sources)                        | Yes                                                                                                               | No. Only query Prometheus-based data sources.                                   |
+| Mix and match data sources                                                                                              | Yes                                                                                                               | No                                                                              |
+| Add [expressions](ref:expression-queries) to transform<wbr /> your data and set [alert conditions](ref:alert-condition) | Yes                                                                                                               | No                                                                              |
+| Use [images in alert notifications](ref:notification-images)                                                            | Yes                                                                                                               | No                                                                              |
+| Support for [recording rules](#recording-rules)                                                                         | Yes                                                                                                               | Yes                                                                             |
+| Organization                                                                                                            | Organize and manage access with folders                                                                           | Use namespaces                                                                  |
+| Alert rule evaluation and delivery                                                                                      | Alert evaluation is done in Grafana, while delivery can be handled by Grafana or an external Alertmanager.        | Alert rule evaluation and alert delivery are distributed.                       |
+| Scaling                                                                                                                 | Alert rules are stored in the Grafana database, which may experience transient errors. It only scales vertically. | Alert rules are stored within the data source and allow for horizontal scaling. |
 
-## Comparison between alert rule types
+## Recording rules
+
+Similar to alert rules, recording rules are evaluated periodically. A recording rule pre-compute frequently used or computationally expensive queries, and saves the results as a new time series metric.
```

Comment:
Similar to alert rules, recording rules are evaluated periodically. A recording rule pre-computes frequently used or computationally expensive queries, and saves the results as a new time series metric.

Reply:
addressed by https://github.com/grafana/grafana/pull/97430/commits/e2cda3deb2d002cece72b767fd3af5c3d3f8da71